configuration.go

```diff
@@ -270,6 +272,12 @@ func NewTraefikDefaultPointersConfiguration() *TraefikConfiguration {
 	defaultDocker.ExposedByDefault = true
 	defaultDocker.Endpoint = "unix:///var/run/docker.sock"
 
+	//default Swarm
+	var defaultSwarm provider.Swarm
+	defaultSwarm.Watch = true
+	defaultSwarm.ExposedByDefault = true
+	defaultSwarm.Endpoint = "unix:///var/run/docker.sock"
```

Comment:
@diegofernandes Are you saying that by binding the /var/run/docker.sock via the manager node, all traffic will be routed through the manager node even if DNS is set up to load balance to the slave nodes? Doesn't the docker socket from the slave node also get binded into the traeffik service container running on the slave nodes @vdemeester @diegofernandes Thanks.